engine/CMakeLists.txt

```diff
@@ -1743,6 +1744,8 @@ IF (USE_GTEST)
 
     TARGET_LINK_LIBRARIES(
         ${TEST_NAME}
+        ${Boost_LIBRARIES}
+        ${Python3_LIBRARIES}
```

Comment:
I wasn't able to make this support python 2, which we still support.

I admit I'm not very good with CMake. We have some code higher up which determines whether we're using ver 2 or 3 but I wasn't able to make it work here.

This is really the show stopper IMO for this PR and where I need your help.

Reply:
0.9.x does not support Python2. No need to be concerned about it.
0.8.x was the last version to support Python 2.